completions/apt-get

```diff
@@ -22,7 +22,7 @@ _comp_cmd_apt_get()
     _comp_initialize -n ':=' -- "$@" || return
 
     local special="" i
-    for ((i = 1; i < ${#words[@]} - 1; i++)); do
+    for ((i = 1; i < cword; i++)); do
```

Comment:
Just a note, the class of these changes makes us more likely complete a "special" when one was already specified after the completion point. It's not necessarily a bad thing, and in some cases we ended up doing that anyway through some possible weird accidents, and in most cases completing something that's valid for a "special" before it isn't the right thing to do anyway. So all good as far as I'm concerned.

Reply:
I also thought about that possibility (that this type of loop is intended for checking also a "special" after the completion point), but in that case, I would expect `i < ${#words[@]}` instead of `i < ${#words[@]} - 1`. Maybe I miss something. Just out of curiosity, do you know or remember the reason for `- 1`? My guess was just that it was written under the assumption that the completion point is always in the last word.

The only exception is `apt-cache` where the case of a "special" after the completion point seems to be explicitly handled. This was processed in a separate commit a57f1b415610af604f275622212573f0231c96de.